package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "bedrock-payment-paypal",
+  "version": "0.0.1",
+  "description": "This Plugin provides PayPal functionality to bedrock-payment.",
```

Comment:
Add license field.

Reply:
Are you sure about this? I thought if there is a License.md in the repo we don't need a license in the package.json? Also what is the license type for the field? Usually use BSD-3-Clause, but this uses the bedrock license.

Reply:
npm warns about no license on install.  And there are tools to recusively report all licenses used in a project via that field.  We've been using what they suggest, which looks absurd and renders poorly on their own site:
https://docs.npmjs.com/files/package.json#license
`"license" : "SEE LICENSE IN LICENSE.md"`

Reply:
done here: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/2/commits/f8d86f95a4c7aeb5508965a7b184d703b8bcf9a9